### DISCUSSION TITLE
Deploying on host with Docker usernamespace isolation turned on

### DISCUSSION THREAD

**Discussion by riccardolardi** [supabase/supabase#26651] [Questions]
Hi supabase
How would you recommend deploying supabase on a machine that has usernamespace isolation turned on for all containers? https://docs.docker.com/engine/security/userns-remap
It will disallow for bind mounts, which is a prerequisite in the docker-compose.yml that is shipped.
Thanks

**Comment by swamidass** [comment]
This is related to [#2356
](https://github.com/supabase/cli/issues/2356)

And also: https://community.atlassian.com/t5/Bitbucket-articles/Changes-to-make-your-containers-more-secure-on-Bitbucket/ba-p/998464
